jsonschema-validation.xml

```diff
@@ -649,12 +635,12 @@
 
             <section title="oneOf">
                 <t>
-                    This keyword's value MUST be an array. This array MUST have at least one
-                    element.
+                    This keyword's value MUST be an array.
+                    This array MUST have at least one element.
                 </t>
                 <t>
-                    Elements of the array MUST be objects. Each object MUST be a valid JSON
-                    Schema.
+                    Elements of the array MUST be objects.
```

Comment:
I thought that's what we did at first and then it was changed back for some reason... idk. I'll do this though.